src/redux/configurestore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
```

Comment:
- [ ] [OPTIONAL] Since you have @reduxjs/toolkit installed, consider using the `configureStore` function instead of the `createStore` as it has some good defaults already. You can read more [here](https://redux-toolkit.js.org/api/configureStore)

Reply:
Thank you @phelian23